components/slot-fill/slot.js

```diff
@@ -54,13 +54,9 @@ class Slot extends Component {
 
 		const fills = map( getFills( name ), ( fill ) => {
 			const fillKey = fill.occurrence;
+			const fillChildren = isFunction( fill.props.children ) ? fill.props.children( fillProps ) : fill.props.children;
```

Comment:
I made this change to allow `null` values.

Reply:
How so? There's still a key added. This change is needed because you can't return null when using the function. See https://github.com/WordPress/gutenberg/pull/6959/files#diff-8c9cd11f122f43914bc769f4ad5c36d9R21.